torchtune/data/_utils.py

```diff
@@ -44,9 +46,9 @@ def truncate(
     return tokens_truncated
 
 
-def load_image(image_loc: Union[Path, str]) -> "PIL.Image.Image":
+def load_image(image_loc: Union[Path, str]) -> "torch.Tensor":
```

Comment:
You don't need to quote the return type as `torch.Tensor` is definitely already importable.